tests/sanity/ignore-2.15.txt

```diff
@@ -8,81 +8,3 @@ plugins/modules/logs_metric_filter.py validate-modules:mutually_exclusive-type
 plugins/modules/wafv2_ip_set.py validate-modules:mutually_exclusive-type
 plugins/modules/wafv2_regex_pattern_set.py validate-modules:mutually_exclusive-type
 plugins/modules/wafv2_web_acl_association.py validate-modules:mutually_exclusive-type
```

Comment:
I guess these ones should not be deleted. And you should also add an entry for the MemoryDB_* modules.